### DISCUSSION TITLE
Mi practica de CSS

### DISCUSSION THREAD

**Discussion by FredYegres** [Launch-X-Latam/MisionFrontEnd#375] [Show and tell]
Aqui les dejo mi practica de CSS no es responsive pero creo que me acerque bastante al resultado final

Enlace al repositorio:
https://github.com/FredYegres/practicas-launch-x/tree/main/Practica-CSS

Enlace al deploy:
https://fredyegres.github.io/practicas-launch-x/Practica-CSS/

**Comment by TonatiuhDSJ** [comment]
orale te quedo super chida.

**Reply by OscarJG20** [reply]
Se ve bastante bien en escritorio, ahora queda el reto personal de la versión móvil. Como recomendación extra, a los botones que aparecen en el apartado de preguntas/respuestas, trata de que la acción de despliegue sea en todo el botón (incluyendo el texto) y no solo al icono, es opcional, pero creo que muchas personas presionarán antes el texto que el icono.
